configs/sdss_standard_ingest.cfg

```diff
@@ -0,0 +1,72 @@
+"""
+This config should be used to ingest SDSS_standards for 'the_monster' catalog
+The data can be found at '/sdf/group/rubin/g/project/the_monster/SDSS' 
+
+config drafted by Peter Ferguson (peter.ferguson@gmail.com)
+
+convertReferenceCatalog ./sharded_refcats ~/u/repos/the_monster/configs/sdss_standard_ingest.cfg '/sdf/home/p/pferguso/u/the_monster/SDSS/*.fit' &> ./sharded_refcats/convert-sdss.log
+
+
+"""
+
+from lsst.meas.algorithms.readFitsCatalogTask import ReadFitsCatalogTask
+
+# Default is ReadTextCatalogTask
+config.file_reader.retarget(ReadFitsCatalogTask)
+
+# String to pass to the butler to retrieve persisted files.
+config.dataset_config.ref_dataset_name='sdss_16_standards_20221205'
+
+
+config.dataset_config.indexer.name='HTM'
+
+# Depth of the HTM tree to make.  Default is depth=7 which gives ~ 0.3 sq. deg. per trixel.
+config.dataset_config.indexer['HTM'].depth=7
+
+# Number of python processes to use when ingesting.
+config.n_processes=5
+
+# Name of RA column
+config.ra_name='ra'
+
+# Name of Dec column
+config.dec_name='dec'
+
+# Name of column to use as an identifier (optional).
+# sdss 'name' is  a string not a int so cannot be used as and identifier 
+# eg. SDSS J114541.01+512803.7
+#config.id_name='name'
```

Comment:
If you want to include this, you should be able to with `extra_col_names`.

Reply:
I tried that. 
if `config.id_name='name'` : ValueError: invalid literal for int() with base 10: b'SDSS J235811.01+104748.3'
if `config.extra_col_names=['name']` : return cls._registry[key], KeyError: <class 'numpy.bytes'>

leaning towards not including 'name' at all.

Reply:
Why is the reader turning that into `bytes`? We could just skip it, but it might be a simple fix in `ReadFitsCatalogTask`, since that's built on `astropy.table.Table.read()`.

Reply:
I was looking at this and I think the current code will fail for any column that is a string. 
`astropy.table.Table.read()` pulls in strings as `<class 'numpy.bytes_'> ` (which works fine)
and the error is getting thrown at `/python/lsst/utils/wrappers.py", line 425, in __getitem__`, specifically 
`cls._registry[key]`
And `cls._registry.keys()` gives 
`dict_keys([<class 'numpy.uint8'>, 'B', <class 'numpy.uint16'>, 'U', <class 'numpy.int32'>, 'I', <class 'numpy.int64'>, 'L', <class 'numpy.float32'>, 'F', <class 'numpy.float64'>, 'D', <class 'str'>, 'String', <class 'lsst.geom.Angle'>, 'Angle', 'ArrayB', 'ArrayU', 'ArrayI', 'ArrayF', 'ArrayD', 'Flag', <class 'float'>])`

The only string related field in this dict is: `'String': <class 'lsst.afw.table.FieldString'>`

Not sure if this is something we should hunt down

Full Traceback:

> Traceback (most recent call last):
  File "/sdf/group/rubin/sw/conda/envs/lsst-scipipe-6.0.0/share/eups/Linux64/meas_algorithms/gf28172b03b+14260db065/bin/convertReferenceCatalog", line 7, in <module>
    main()
  File "/sdf/group/rubin/sw/conda/envs/lsst-scipipe-6.0.0/share/eups/Linux64/meas_algorithms/gf28172b03b+14260db065/python/lsst/meas/algorithms/convertReferenceCatalog.py", line 681, in main
    run_convert(args.outputDir, args.configFile, args.fileglob[0])
  File "/sdf/group/rubin/sw/conda/envs/lsst-scipipe-6.0.0/share/eups/Linux64/meas_algorithms/gf28172b03b+14260db065/python/lsst/meas/algorithms/convertReferenceCatalog.py", line 661, in run_convert
    converter.run(files)
  File "/sdf/group/rubin/sw/conda/envs/lsst-scipipe-6.0.0/share/eups/Linux64/meas_algorithms/gf28172b03b+14260db065/python/lsst/meas/algorithms/convertReferenceCatalog.py", line 493, in run
    schema, key_map = self._writeMasterSchema(inputFiles[0])
  File "/sdf/group/rubin/sw/conda/envs/lsst-scipipe-6.0.0/share/eups/Linux64/meas_algorithms/gf28172b03b+14260db065/python/lsst/meas/algorithms/convertReferenceCatalog.py", line 600, in _writeMasterSchema
    schema, key_map = self.makeSchema(arr.dtype)
  File "/sdf/group/rubin/sw/conda/envs/lsst-scipipe-6.0.0/share/eups/Linux64/meas_algorithms/gf28172b03b+14260db065/python/lsst/meas/algorithms/convertReferenceCatalog.py", line 588, in makeSchema
    key_map[col] = addField(col)
  File "/sdf/group/rubin/sw/conda/envs/lsst-scipipe-6.0.0/share/eups/Linux64/meas_algorithms/gf28172b03b+14260db065/python/lsst/meas/algorithms/convertReferenceCatalog.py", line 585, in addField
    return schema.addField(name, at_type)
  File "/sdf/group/rubin/sw/conda/envs/lsst-scipipe-6.0.0/share/eups/Linux64/afw/g2c743f43ab+04f223c4f0/python/lsst/afw/table/_schema.py", line 160, in addField
    field = Field[type](field, doc=doc, units=units,
  File "/sdf/group/rubin/sw/conda/envs/lsst-scipipe-6.0.0/share/eups/Linux64/utils/ga34f01d9d8+f0a1800e64/python/lsst/utils/wrappers.py", line 425, in __getitem__
    return cls._registry[key]
KeyError: <class 'numpy.bytes_'>

Reply:
I think that we should drop the SDSS id for now (we can always add it later if we need it), so that we can merge these configs.